pr-temp/Where-To-Put-Documentation.md

```diff
@@ -0,0 +1,119 @@
+## Overview
+
+As with all large projects, the Fluid Framework requires a sizable amount of documentation.
+That documentation lives in a number of different locations / systems.
+It can sometimes be difficult to find the documentation you need.
+Similarly, it can sometimes be hard to determine where new documentation should live.
+
+This document aims to give simple guidance to make our documentation systems easier to navigate, and to ensure we have consistent policy with regards to where different forms of documentation should go.
+
+## Documentation locations
+
+The following are locations where Fluid Framework documentation can be found.
+They are not presented in any particular order.
+
+- [fluidframework.com](https://fluidframework.com/)
+  - This is our customer-facing website.
+    It is the public face of our product and is home to high-level overviews, samples, etc. to get users up-and-running on the framework.
+- [Github Wiki](https://github.com/microsoft/FluidFramework/wiki)
+  - This is the contributor-facing wiki for the [fluid-framework](https://github.com/microsoft/FluidFramework) repository.
+- (Microsoft Internal) [EngHub Wiki](https://eng.ms/docs/experiences-devices/oxo/office-shared/wacbohemia/fluid-framework-platform-internal/ff-platform-docs/docs/overview)
```

Comment:
It's an interesting question. I do think most of these guidelines should be visible to all of our contributors. The MS-specific guidelines perhaps not. I think the trade off is either having a single point of entry that has some labeled links that are MS-internal, or having 2 separate pages. I was leaning towards the former, but I am definitely open to other perspectives here 🙂

Reply:
I guess even if we did use the latter pattern, I would probably at least want to have a disclaimer about / link to the MS-internal guidelines for awareness. But it would still reduce the number of elements on the page that are not relevant to our open-source community.

Reply:
May be external and internal can have different point of entry. but the internal info points to the external information to avoid duplication?  We sync the Github wiki content to the internal doc as well, so they can be referenced that way as well.

Reply:
That seems reasonable to me. I still like the idea of having at least a quick disclaimer in the github wiki doc pointing to the internal document in case an internal developer discovers the github wiki article first without knowing about the internal variant. Does that seem reasonable to you?

Reply:
@curtisman I took a stab at splitting the document into 2 - one for the EngHub wiki and one for the github wiki.

I left a brief section in the github wiki doc for Microsoft devs reminding them **not** to put team/company/partner-internal docs anywhere public, but otherwise the contents are community-facing. And the EngHub page defers to the github doc where appropriate to avoid some redundancy.

Let me know what you think :)